vms/platformvm/health.go

```diff
@@ -16,6 +16,9 @@ func (vm *VM) Health() (interface{}, error) {
 	if err != nil {
 		return nil, fmt.Errorf("couldn't get percent connected: %w", err)
 	}
+
+	vm.metrics.percentConnected.Add(percentConnected)
```

Comment:
Shouldn't this be `.Set` rather than `.Add`?

Reply:
Good catch!